dev/io.openliberty.checkpoint/resources/io/openliberty/checkpoint/resources/CheckpointMessages.nlsprops

```diff
@@ -37,3 +37,22 @@ RESTORE_FAILED_CWWKC0454E=CWWKC0454E: The server failed to restore from a checkp
 RESTORE_FAILED_CWWKC0454E.explanation=An attempt to restore the server from a checkpoint failed.  The server is not running.
 RESTORE_FAILED_CWWKC0454E.useraction=Inspect the checkpoint directory located in the server logs directory for additional log files to determine why the server could not be restored.
 
+UKNOWN_FAILURE_CWWKC0455E=CWWKC0455E: An unknown error occurred with the following message: {0}
+UKNOWN_FAILURE_CWWKC0455E.explanation=An unexpected error occurred during a checkpoint request or during an attempt to restore the server from a checkpoint.
+UKNOWN_FAILURE_CWWKC0455E.useraction=Inspect the checkpoint directory located in the server logs directory for additional log files to determine why the error occurred.
+
+CHECKPOINT_FAILED_UNSUPPORTED_FEATURE_CWWKC0456E=CWWKC0456E: A checkpoint cannot be taken because the following features configured in the server.xml file are not supported for checkpoint: {0}
+CHECKPOINT_FAILED_UNSUPPORTED_FEATURE_CWWKC0456E.explanation=The server checkpoint action can only be used with features that support checkpoint.  
+CHECKPOINT_FAILED_UNSUPPORTED_FEATURE_CWWKC0456E.useraction=Only enable features that support checkpoint to use a server checkpoint action. Check the documentation for details on the features that support checkpoint.
+
+CHECKPOINT_FAILED_PREPARE_EXCEPTION_CWWKC0457E=CWWKC0457E: A error occurred preparing to take a checkpoint with the following message: {0}
```

Comment:
Please provide more info in the message. What caused the error? Did the error stop preparations for checkpoint?

Nit: "A error" (add "n")

Reply:
I'll fix up the need for `An error`.  See my previous message about the context of these messages.  More details are provided by the proceeding and appending messages to this one.